src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -197,21 +197,41 @@ cdef class _ServicerContext:
     def set_trailing_metadata(self, object metadata):
         self._rpc_state.trailing_metadata = tuple(metadata)
 
+    def get_trailing_metadata(self):
+        return self._rpc_state.trailing_metadata
+
+    trailing_metadata = property(get_trailing_metadata, set_trailing_metadata)
```

Comment:
`property` is a nice Python3 feature, but gRPC Python code base isn't using it yet. If we decided to use it, we should add it unanimously to all interfaces.